client/resources.js

```diff
@@ -14,6 +14,15 @@ const storyFactory = (args: Object): StoryType => ({
   ...args,
 })
 
+export type GroupType = {
+  uid: string,
+}
+
+const groupFactory = (args: Object): GroupType => ({
```

Comment:
Groups won't need factories since they're all created on the server (unlike stories and clues etc.) This can be removed.